Dockerfile

```
@@ -35,14 +35,8 @@ RUN <<EOF
     rm -rf $VOICEVOX_CORE_ASSET_NAME
     rm "./${VOICEVOX_CORE_ASSET_NAME}.zip"
 
-    # Move Core Library to /opt/voicevox_core/
-    mkdir /opt/voicevox_core
-    mv "./core/libcore.so" /opt/voicevox_core/
-
-    # Move documents to /opt/voicevox_core/
-    mv ./core/VERSION /opt/voicevox_core/
-
-    rm -rf ./core
+    # Move Core to /opt/voicevox_core/
+    mv ./core/ /opt/voicevox_core/
```

- `/opt/voicevox_core/` ディレクトリが事前に作られていないため、エラーになりそうでした。
- `mv ./core/ /opt/voicevox_core/` だと `/opt/voicevox_core/core/...` というディレクトリの構造になると思いますが、変更前の処理を見るに `/opt/voicevox_core/libvoicevox_core.so` のように `core/` ディレクトリをはさまないような構造を想定していると思いました。`mv ./core/* /opt/voicevox_core/` として、`core/` ディレクトリ以下のファイルが直接 `/opt/voicevox_core/` ディレクトリに展開されるようにすると良さそうでした。

```suggestion
    # Move Core to /opt/voicevox_core/
    mkdir /opt/voicevox_core
    mv ./core/* /opt/voicevox_core/
```

---

ありがとうございます！！
これ（`mv ./core/ /opt/voicevox_core/`）、やってることは`./core/`ディレクトリを`/opt/voicevox_core/`にリネームなので、むしろディレクトリが存在しないことで正しい挙動になってる感じです！

でもこれ`/opt/voicevox_core/`のディレクトリ有無で挙動が変わるのでかなり危ないなと思いました。
なので提案されている形のほうが良さそうです！！

---

> ありがとうございます！！ これ（`mv ./core/ /opt/voicevox_core/`）、やってることは`./core/`ディレクトリを`/opt/voicevox_core/`にリネームなので、むしろディレクトリが存在しないことで正しい挙動になってる感じです！

すみません、意図を勘違いして、内容的にも間違った指摘をしていました。説明ありがとうございます！　問題ありませんでしたね。

> でもこれ`/opt/voicevox_core/`のディレクトリ有無で挙動が変わるのでかなり危ないなと思いました。 なので提案されている形のほうが良さそう！！

こちら了解です！